docs/installation/native-installation.md

```diff
@@ -52,6 +53,25 @@ monolog:
       path: "%kernel.logs_dir%/%kernel.environment%.cron.log"
 ```
 
+#### 2.2. Set up the microservice for product search
+For the product search to be working correctly, you'll have to install a [microservice for product search](https://github.com/shopsys/microservice-product-search).
+It will act as a fully independent unit with a separate web server and repository.
+
+Just clone the repository into a separate directory, install all its dependencies, configure connection to the Postgres by setting up the PG_CONNECTION_STRING environment variable (or the [.env file](http://symfony.com/doc/current/components/dotenv.html)) and run the server.
+In the current version of this experimental microservice, the PHP's built-in Web Server is used.
+
+```
+git clone https://github.com/shopsys/microservice-product-search.git
+cd microservice-product-search
+composer install
+# open ".env" and set up connection to Postgres in "PG_CONNECTION_STRING" 
```

Comment:
this comment is weird, it says:
php bin/console server:run 127.0.0.1:8001 is doing open ".env" and set up connection to Postgres in "PG_CONNECTION_STRING" 

This needs to be reworked

Reply:
Im not sure, I dont see "php bin/console server:run 127.0.0.1:8001 is doing open"? Can you be more specific? Thank you

Reply:
I see now, these commands will be splitted

Reply:
We have discussed this